cli/src/main.rs

```diff
@@ -359,6 +381,26 @@ fn main() {
             println!("{}", jwk_str);
         }
 
+        DIDKit::Key(cmd) => match cmd {
+            KeyCmd::Generate(cmd_generate) => {
+                let jwk_str = match cmd_generate {
+                    KeyGenerateCmd::Ed25519 => {
+                        let jwk = JWK::generate_ed25519().unwrap();
+                        serde_json::to_string(&jwk).unwrap()
+                    }
+                    KeyGenerateCmd::Secp256k1 => {
+                        let jwk = JWK::generate_secp256k1().unwrap();
+                        serde_json::to_string(&jwk).unwrap()
+                    }
+                    KeyGenerateCmd::Secp256p1 => {
```

Comment:
https://github.com/spruceid/didkit/pull/259/files#r803829151
```suggestion
                    KeyGenerateCmd::Secp256r1 => {
```